en/publish/publish-qutoutiao-mini-games.md

```diff
@@ -81,6 +75,12 @@ After the relevant parameters of the **Build** panel are set, click **Build**. A
 
 - After setting, click the **打开游戏** button below to open the game. **Note**: If you want to open the game again, you need to re-place the cpk package in the file path described above.
 
+## Resource Management for QTT Mini Game Environment
+
+**QTT Mini Game** is similar to **WeChat Mini Game**. There are restrictions on the package size. Resources over **4MB** must be downloaded via a network request.
```

Comment:
Resources over **4MB** must be downloaded via a network request. -> The main package size limit for QTT Mini Game is **4MB**, more than that must be downloaded via a network request.